content/en/networks/calibration/explorers/index.md

```diff
@@ -15,8 +15,7 @@ toc: true
 
 ## Filscout
 
-[calibration.filscout.com](https://calibration.filscout.com/en)
```

Comment:
Both https://calibration.filscout.com/en and https://calibration.filscout.com are returning a 502 Bad Gateway error. Removing until we figure out what's up. 

@johnnymatthews